src/Upgrade/Application/Strategy/ReleaseApp/Processor/SequentialReleaseGroupProcessor.php

```diff
@@ -159,7 +159,6 @@ public function process(ReleaseGroupDtoCollection $requireRequestCollection, Ste
             }
 
             if (!$response->isSuccessful()) {
-                $stepsExecutionDto->setIsSuccessful(false);
```

Comment:
if it is a blocker, then it should be failed, isn't it?

Reply:
It's blocker for specific RG, that's why we brake the cycle. But for previous commits should be saved